repository/repository-server-config/pom.xml

```diff
@@ -80,7 +87,21 @@
 			<artifactId>commons-compress</artifactId>
 			<version>1.18</version>
 		</dependency>
-
+		
+		<!-- AWS Elastic Search configuration (start) -->
+		<dependency>
+		    <groupId>com.amazonaws</groupId>
+		    <artifactId>aws-java-sdk-core</artifactId>
+		    <version>1.11.574</version>
+		</dependency>
+		
+		<dependency>
+		    <groupId>com.github.awslabs</groupId>
```

Comment:
I cannot find any GitHub repository with the sources for the specified version. Can you please comment in the PR where you get it from ?

Reply:
This is what I talked about in the email I sent last friday on vorto-dev. This artifact is not released in Maven central, hence the jitpack workaround.

It comes from this repo.

https://github.com/awslabs/aws-request-signing-apache-interceptor

Jitpack

https://jitpack.io/#awslabs/aws-request-signing-apache-interceptor

The version, I think, is lifted from the last github commit signature. Jitpack builds artifacts on demand.

The alternative to using this is to actually copy the file and put it in our project. I think we'll have an IP problem with that approach.